Ginger/GingerCoreTest/AutoPilot/APIModelsCompareTest.cs

```diff
@@ -0,0 +1,185 @@
+﻿using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.Repository.ApplicationModelLib;
+using Amdocs.Ginger.CoreNET.Repository;
+using Amdocs.Ginger.Repository;
+using Ginger.Run;
+using GingerCore.Drivers.WebServicesDriverLib;
+using GingerCore.Platforms;
+using GingerCoreNET.Application_Models;
+using GingerCoreNET.SolutionRepositoryLib.RepositoryObjectsLib.PlatformsLib;
+using GingerTestHelper;
+using GingerWPF.ApplicationModelsLib.APIModels;
+using GingerWPF.WorkSpaceLib;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UnitTests.NonUITests.AutoPilot
+{
+    /// <summary>
+    /// API Models Comparison Utility Test
+    /// </summary>
+    /// 
+
+    [TestClass]
+    public class APIModelsCompareTest
+    {
+        static GingerRunner mGR = null;
+
+        static List<string> xmlFiles;
+        static List<ApplicationAPIModel> learnedAPIsList;
+        static ObservableList<ApplicationAPIModel> existingAPIsList;
+        static int textExecutedCount = 0;
+
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext TC)
+        {
+            Reporter.ToLog(eLogLevel.DEBUG, "Creating the GingerCoreNET WorkSpace");
+            WorkSpaceEventHandler WSEH = new WorkSpaceEventHandler();
+            WorkSpace.Init(WSEH);
+
+            string TempSolutionFolder = TestResources.GetTestTempFolder(@"Solutions" + Path.DirectorySeparatorChar + "APIModelsComparisonUtilityTest");
+            if (Directory.Exists(TempSolutionFolder))
+            {
+                Directory.Delete(TempSolutionFolder, true);
+            }
+
+            // Initialize XML Parser 
+            XMLTemplateParser xmlParser = new XMLTemplateParser();
+
+            // Initialize XML file names to be fetched from TestResources
+            xmlFiles = new List<string>()
+            {
+                @"XML\API_Templates_XML\Create_User.xml",
+                @"XML\API_Templates_XML\Delete_User.xml",
+                @"XML\API_Templates_XML\PhoneVerifySOAP_CheckPhoneNumber.xml",
```

Comment:
I don't see "API_Templates_XML" folder in the project files tree maybe that what causing the unit tests to fail because resource files failed to be found

Reply:
Updated !